e2e/test_playwright.py

```diff
@@ -0,0 +1,370 @@
+import pytest
+import time
+from subprocess import Popen, PIPE
+from http.client import HTTPConnection
+
+from piccolo_admin.example import Director
+from piccolo_admin.example_data import DIRECTORS
+
+from playwright.sync_api import Playwright, sync_playwright
+
+
+@pytest.fixture
+def dev_server():
+    """
+    Running dev server and Playwright test in parallel.
+    More info https://til.simonwillison.net/pytest/playwright-pytest
+    """
+    process = Popen(
+        ["python", "-m", "piccolo_admin.example"],
+        stdout=PIPE,
+    )
+    retries = 5
+    while retries > 0:
+        conn = HTTPConnection("localhost:8000")
+        try:
+            conn.request("HEAD", "/")
+            response = conn.getresponse()
+            if response is not None:
+                yield process
+                break
+        except ConnectionRefusedError:
+            time.sleep(1)
+            retries -= 1
+
+    Director.delete(force=True).run_sync()
+    Director.insert(*[Director(**d) for d in DIRECTORS]).run_sync()
```

Comment:
We might not need to do this anymore - it looks like each test starts the server again, so it will already have fresh data each time.

Reply:
Let's keep Storybook for now - otherwise this PR gets quite big.

Maybe we drop it in the future. I'm torn on Storybook. It can be useful during development sometimes, and encourages us to write nicely isolated components. But it's a pain to configure using Webpack.